test/v4/expectedRate.js

```diff
@@ -425,15 +426,15 @@ contract('ExpectedRate', function(accounts) {
     });
 
     it("should verify get expected rate reverts when qty * qtyFactor > MAX QTY.", async function() {
-        let legalQty = (new BigNumber(10).pow(28)).div(2);
-        let illegalQty = (new BigNumber(10).pow(28)).div(2).add(1);
+        let legalQty = (new BN(10).pow(new BN(28))).div(new BN(2));
+        let illegalQty = (new BN(10).pow(new BN(28))).div(new BN(2)).add(new BN(1));
```

Comment:
for next files.
try shorter sequence
ex:
new BN((10 ** 18) + 1);
instead of this line.